src/Map/src/DependencyInjection/UXMapExtension.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\DependencyInjection;
+
+use Symfony\Component\AssetMapper\AssetMapperInterface;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ChildDefinition;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
+use Symfony\Component\DependencyInjection\Loader\PhpFileLoader;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+use Symfony\UX\Map\Bridge as MapBridge;
+
+/**
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+class UXMapExtension extends Extension implements ConfigurationInterface, PrependExtensionInterface
+{
+    /**
+     * @var array<string, array{ package: class-string<MapBridge\AbstractPackage> }>
+     */
+    private static array $bridges = [
+        'google' => [
+            'package' => MapBridge\Google\Package::class,
+        ],
+        'leaflet' => [
+            'package' => MapBridge\Leaflet\Package::class,
+        ],
+    ];
+
+    public function load(array $configs, ContainerBuilder $container): void
+    {
+        $config = $this->processConfiguration($this, $configs);
+
+        $loader = (new PhpFileLoader($container, new FileLocator(\dirname(__DIR__).'/../config')));
+        $loader->load('services.php');
+
+        $renderers = ['main' => $config['renderer']];
+        $container->getDefinition('ux_map.renderers')->setArgument(0, $renderers);
+
+        foreach (self::$bridges as $name => ['package' => $package]) {
+            if (ContainerBuilder::willBeAvailable('symfony/ux-map-'.$name, $package, ['symfony/ux-map'])) {
```

Comment:
Not sure you should pass the third parameter

https://github.com/symfony/symfony/blob/e27db25e4e4291a3109822969b780e58528fa2a1/src/Symfony/Component/DependencyInjection/ContainerBuilder.php#L1494C8-L1495C83

Reply:
TBH I don't 100% understand this code, I've followed what it's done for [notifiers](https://github.com/symfony/symfony/blob/7.2/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L2848), [mailers](https://github.com/symfony/symfony/blob/7.2/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L2644-L2646)